reo/scenario.py

```diff
@@ -199,33 +199,50 @@ def setup_pv(pv_dict, latitude, longitude, time_steps_per_hour):
         lp.load_list = [0 if ((x > negative_load_tolerance) and (x < 0)) else x for x in lp.load_list]
 
         # Load Profile Boiler Fuel
-        lpbf_space = LoadProfileBoilerFuel(load_type="SpaceHeating",
-            dfm=dfm,
-            time_steps_per_hour=inputs_dict['time_steps_per_hour'],
-            latitude=inputs_dict['Site']['latitude'],
-            longitude=inputs_dict['Site']['longitude'],
-            nearest_city=lp.nearest_city,
-            year=lp.year,
+        lpbf_space = LoadProfileBoilerFuel(
```

Comment:
Why are there changes in the reo directory, which only affects the discontinued v1-2?